commit 6325d19525d71051ccbd1dc2a030a0839c95c7d9
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Sun Feb 12 19:48:26 2017 +0100

    Drop search for obsolete Libkcddb, don't be quiet about KF5Cddb
    
    KF5-based releases are found by KF5Cddb since 16.12.0.
    
    REVIEW: 129945

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2b286cf..893b2d6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -128,18 +128,11 @@ if(KDEPIMLIBS_FOUND)
 endif(KDEPIMLIBS_FOUND)
 
 # There is a port of libkcddb to use KF5 style linking and headers
-find_package(KF5Cddb QUIET)
+find_package(KF5Cddb)
 set_package_properties(KF5Cddb PROPERTIES
     DESCRIPTION "Support for CDDB searches"
-    URL "http://projects.kde.org/projects/kde/kdemultimedia/libkcddb"
+    URL "https://cgit.kde.org/libkcddb.git"
     TYPE OPTIONAL)
-if(NOT KF5Cddb_FOUND)
-  find_package(Libkcddb 5.0)
-  set_package_properties(Libkcddb PROPERTIES
-      DESCRIPTION "Support for CDDB searches"
-      URL "http://projects.kde.org/projects/kde/kdemultimedia/libkcddb"
-      TYPE OPTIONAL)
-endif(NOT KF5Cddb_FOUND)
 
 find_package(Taglib)
 set_package_properties(Taglib PROPERTIES
